src/templates/app/docker-compose.yml.ecr

```diff
@@ -0,0 +1,79 @@
+version: '2'
+<% if @database == "pg" %>
+services:
+  web:
+    build: .
+    image: kemalyst-app
+    command: 'kgen watch'
+    working_dir: /app/user
+    environment:
+      PORT: 3000
+      DATABASE_URL: 'postgres://postgres:password@pg:5432/postgres'
+    ports:
+      - '3000:3000'
+    depends_on:
+      - pg
+    volumes:
+      - '.:/app/user'
+
+  migrate:
+    image: kemalyst-app
+    command: 'kgen migrate up'
+    working_dir: /app/user
+    environment:
+      DATABASE_URL: 'postgres://postgres:password@pg:5432/postgres'
+    volumes:
+      - '.:/app/user'
+    depends_on:
+      - pg
+
+  pg:
+    image: postgres
+    environment:
+      POSTGRES_PASSWORD: 'password'
+    volumes:
+      - 'pg:/var/lib/postgres/data'
+
+volumes:
+  pg:
+<% elsif @database == "mysql" %>
+services:
```

Comment:
I'm wondering if we can get rid of this duplicate code.  Only the db's and url's change.